pkg/common/common.go

```diff
@@ -28,14 +29,14 @@ func GetNamespaceSelectorsFromGenericInformer(kind, namespaceOfResource string,
 	namespaceLabels := make(map[string]string)
 	if kind != "Namespace" {
```

Comment:
Why are we checking kind here? Isn't the target resource Namespace?

Reply:
the resource coming in `GetNamespaceSelectorsFromGenericInformer` or in `GetNamespaceSelectorsFromNamespaceLister` function is not namespace. It is the resource on which policy is applied. 

I have added the extra check here because in case in the above function if the resource kind is namespace, the value assigned to the field namespace will be  empty string or `\` which gives error.